torch/ao/quantization/fx/prepare.py

```diff
@@ -1377,7 +1377,35 @@ def insert_observers_for_model(
                         node_name_to_target_dtype_info, node_name_to_qconfig,
                         model, modules, graph)
 
+        # Second pass: Look for getitem nodes and make the input and output observers the same.
         #
+        # Context: There are two use cases of getitem:
```

Comment:
should we move this to the doc for `_make_getitem_share_input_output_observers`?